pwnlib/elf/elf.py

```diff
@@ -530,6 +530,8 @@ def address(self, new):
         self.symbols = dotdict({k:update(v) for k,v in self.symbols.items()})
         self.plt     = dotdict({k:update(v) for k,v in self.plt.items()})
         self.got     = dotdict({k:update(v) for k,v in self.got.items()})
+        for f in self.functions.values():
```

Comment:
Can you use the `dotdict` update function like the rest of the values?

Thanks!

Reply:
I'm not sure what you mean by the `dotdict` update function. The lambda right above?
All the other dicts are just `name=>address` maps, but the `functions` dict is `name=>Function object`, so I don't want to create new `Function` objects, just change one member attribute of those instances.
Is there a nicer way to do this?

Reply:
LGTM. If `Function` is mutable, I am for. Recreating it would be just a waste of memory allocations.